docs-src/concepts/what-is-worker-versioning.md

```diff
@@ -8,6 +8,12 @@ tags:
   - versioning
 ---
 
+:::warning
```

Comment:
```suggestion
:::info
```
The `warning` admonition is limited to actions that can cause data loss or other irreversible damage.